base/deprecated.jl

```diff
@@ -1595,6 +1595,8 @@ end
 # issue #6466
 # `write` on non-isbits arrays is deprecated in io.jl.
 
+@deprecate isconcrete isconcrete
```

Comment:
`@deprecate isleaftype isconcrete`? Or are you trying to deprecate what you just introduced?